### PR TITLE
Register xkotelek.is-a.dev

### PR DESCRIPTION
**Pull request by xKotelek**
Register xkotelek.is-a.dev with CNAME record pointing to xkotelek.github.io.

### CODE REVIEW

File: CNAME
Comment (CuteDog5695) [inline CNAME:1]: Delete this file please

**Review by iostpa — CHANGES_REQUESTED** [review]

File: CNAME
Comment (xKotelek) [reply CNAME:1]: what file?

Comment (xKotelek) [reply CNAME:1]: kotelek.json was deleted

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Comment by xKotelek** [comment]
Still can't enforce HTTPS 
image
And an error:
image
Connection is not private
Attackers may be trying to steal your information from xkotelek.is-a.dev (such as passwords, messages, or credit card information). More information
net::ERR_CERT_COMMON_NAME_INVALID
xkotelek.is-a.dev typically uses encryption to protect your information. When Brave tried to connect to xkotelek.is-a.dev this time, it sent back unusual and incorrect login information. This can happen when an attacker is impersonating xkotelek.is-a.dev or when the Wi-Fi login screen interrupts the connection. Your information is still safe because the connection in Brave ended before any data was exchanged.

You can't visit xkotelek.is-a.dev right now because it uses HSTS. Network errors and attacks are usually temporary, so the site will likely be available later.

**Pull request merged**